### PR TITLE
[Console] Remove deprecated console.exception event

### PR DESCRIPTION
**Pull request by mbabker**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
missing CHANGELOG updates in components (please check any your other PRs also if any)

**Comment by mbabker** [comment]
Added

**Review by chalasr — APPROVED** [review]

File: src/Symfony/Component/Console/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+4.0.0
+-----
+
+* [BC BREAK] removed `ConsoleExceptionEvent`
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Console/CHANGELOG.md]: no need for these, we did add them in other PRs

Comment (nicolas-grekas) [inline src/Symfony/Component/Console/CHANGELOG.md]: (I mean the BC BREAL prefix)

Comment (mbabker) [inline src/Symfony/Component/Console/CHANGELOG.md]: Whatever PR I copied from had it, so that's what I rolled with.  Rebased and removed the prefix.

**Comment by fabpot** [comment]
Thank you @mbabker.

**Pull request merged**